ide/ide.kit/test/coverage-manual/build.xml

```diff
@@ -67,7 +67,7 @@
     
     <target name="define-other-tasks">
         <ant dir="selectjars" antfile="build.xml" target="jar"/>
-        <taskdef name="select-jars" classname="selectjars.SelectJars" 
+        <taskdef name="ide/select-jars" classname="selectjars.SelectJars" 
```

Comment:
this taskdef name change may cause problems. see the other [comment](https://github.com/apache/incubator-netbeans/pull/791/#pullrequestreview-149510891) for details